### ISSUE TITLE
PyPI

### ISSUE THREAD

**Issue by jayvdb** [nlohmann/swagger_to_uml#26]
This tool doesnt appear to be on PyPI yet.

**Comment by nlohmann** [comment]
No, it's not. Development has stopped right now, so this is not on my agenda at the moment.
